packages/react-sdk/README.md

```diff
@@ -691,7 +685,7 @@ Run `yarn dev` to build the SDK on changes and launch Storybook.
 
 - `yarn build`: Builds the SDK
 - `yarn clean`: Removes `node_modules`, `lib`, and `.turbo` folders
-- `yarn dev`: Launches Storybook for SDK components
+- `yarn dev`: Launches Storybook for SDK components. Also launches the React Quickstart example app at `http://127.0.0.1:5173/`.
```

Comment:
@rygine - I have a few questions about the `yarn dev` command when run in different directories. =) I wonder if I'm misunderstanding the command? Thank you....

- When I run `yarn dev` in `xmtp-web` - what should I expect? What I see is that it starts the example app at http://127.0.0.1:5173/, but does it also start Storybook? It doesn't automatically open a Storybook browser tab for me - should it? What port is it running on?

- When I run `yarn dev` in `xmtp-web/packages/react-sdk` - what should I expect? It doesn't open Storybook and I don't see the example app running.

- When I run `yarn dev` in `xmtp-web/examples/react-quickstart` - it starts the example app at http://127.0.0.1:5173/.